database/program_db.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
         end_date: programObj.end_date,
         min_grade_level: programObj.min_grade_level,
         max_grade_level: programObj.max_grade_level,
-        program_id_number: `${programObj.title}.${ await module.exports.getTitleCount(programObj.title)}`,
+        program_id: `${programObj.title}.${ await module.exports.getTitleCount(programObj.title)}`,
+        student_list: programObj.student_list,
```

Comment:
Like above, when storing the `student_list`, you'll also want to access each Student object and add this program to their `program_list`. Or you could use a linked table/join table type structure.

Reply:
Again similar to above - you'll also want to update the editProgramById function to do the same thing - update any students that have changed. If a Program has removed a Student, it should be removed from that Student's `program_list`. If a Program has added a Student, that Program should be added to the Student's `program_list`.